Geometry2d/include/Geometry2d/Line.hpp

```diff
@@ -77,23 +72,13 @@ class Line {
         return (d.x() * v.y() - v.x() * d.y());
     }
 
-    /**
-     * @brief Implicit conversion to LineMsg
-     * @return
-     */
-    [[nodiscard]] operator LineMsg() const {
-        LineMsg msg{};
-        msg.pt = {pt[0], pt[1]};
-        return msg;
-    }
-
     /**
      * Returns the point on the line closest to p.
      */
     Point nearestPoint(Point p) const;
 
     /** the line consists of two points */
-    Point pt[2];
+    std::array<Point, 2> pt;
```

Comment:
consider `std::pair` to avoid the complexity of arrays, e.g. out of bounds errors

Reply:
This is not intended to do any significant refactoring on `Line`; we're just bringing it into this decade (real reason is I didn't want to do a version templatized over regular fixed-length arrays).

Reply:
This is probably a good suggestion for a future cleanup effort, though.